### ISSUE TITLE
Add a way to send update notifications to the official Python events calendar project

### ISSUE THREAD

**Issue by malemburg** [python-organizers/conferences#153]
As you may know, there is already a team of people maintaining the official Python events calendars, which are then displayed on python.org/events/:

- https://wiki.python.org/moin/PythonEventsCalendar

To avoid duplicate work for conference organizers, would it be possible to send new submissions in our email format (see wiki page) to our team for inclusion in the calendars ?

Thanks.

**Comment by Mariatta** [comment]
Yeah I think it's possible to write a GitHub action that sends email whenever there's a new commit in the csv. But someone gotta write the code to do it :)

**Comment by Mariatta** [comment]
Before we go further, how does events got added to the calendar from the email? Is it done manually by adding to the calendar? Is it a Google Calendar kind of thing? Is it possible for us to just automatically add to that calendar instead of sending an email?

**Comment by malemburg** [comment]
On 12.08.2020 20:52, Mariatta wrote:
> Before we go further, how does events got added to the calendar from the
> email? Is it done manually by adding to the calendar? Is it a Google
> Calendar kind of thing? Is it possible for us to just automatically add
> to that calendar instead of sending an email?

The process is described on the wiki page. We request the additions
in a special format, this is then checked manually and added manually
to the calendars.

It is not possible to automate this, since we have to run quality
checks on the entries and also select the right calendar for each
event.

-- 
Marc-Andre Lemburg
http://www.malemburg.com/

**Comment by Mariatta** [comment]
Ok. I might handle this by using webhooks and Zapier to send email, instead of GitHub Actions.
Quick look at several GitHub Actions for sending emails, they all seem like wanting me to put in email username and password as secret values, but I'm not comfortable with that approach.
